pynsxv/library/libutils.py

```diff
@@ -95,12 +95,13 @@ def get_scope(client_session, transport_zone_name):
     try:
         vdn_scopes = client_session.read('vdnScopes', 'read')['body']
         vdn_scope_list = client_session.normalize_list_return(vdn_scopes['vdnScopes'])
-        vdn_scope = [scope['vdnScope'] for scope in vdn_scope_list
-                     if scope['vdnScope']['name'] == transport_zone_name][0]
+	vdn_scope = [scope['vdnScope'] for scope in vdn_scope_list
+			if scope['vdnScope'][0]['name'] == transport_zone_name][0]
```

Comment:
Here's what happens without my change:

Traceback (most recent call last):
  File "cli.py", line 66, in <module>
    main()
  File "cli.py", line 62, in main
    args.func(args)
  File "/home/cc-admin/grbpynsxv/py2/pynsxv/pynsxv/library/nsx_logical_switch.py", line 216, in _lswitch_main
    logical_switch_name=args.name, verbose=args.verbose)
  File "/home/cc-admin/grbpynsxv/py2/pynsxv/pynsxv/library/nsx_logical_switch.py", line 74, in _logical_switch_create
    logical_switch_name)
  File "/home/cc-admin/grbpynsxv/py2/pynsxv/pynsxv/library/nsx_logical_switch.py", line 48, in logical_switch_create
    vdn_scope_id, vdn_scope = get_scope(client_session, transport_zone)
  File "/home/cc-admin/grbpynsxv/py2/pynsxv/pynsxv/library/libutils.py", line 99, in get_scope
    if scope['vdnScope']['name'] == transport_zone_name][0]
TypeError: list indices must be integers, not str


you can repro with "pynsxv lswitch create -n something"